app/auth/error/page.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { Fragment } from "react";
+import { useSearchParams } from "next/navigation";
+
+import { signIn } from "next-auth/react";
+
+/** Custom sign-in error page, see documentation at https://next-auth.js.org/configuration/pages */
+const Page = () => {
+  const params = useSearchParams();
+  const error = params.get("error");
+  const Code = ({ children }: { children: React.ReactNode }) => (
```

Comment:
Stojí tohle opravdu za "samostatnou" komponentu? Jakože je to daleko víc textu, než kdybys jen v tom textu níže použil `<code className="bg-yellow">{error}</code>`, ale možná to má nějakej spešl důvod, kterej mi nedotejká. 🙂

Reply:
Haha! Já jsem tam původně měl několik použití, takže se to vyplatilo. Ale nakonec jsem to tam nechal i s tím jedním použitím hlavně kvůli čitelnosti výrazu `{error ? <Code>{error}</Code> : "neznámý"}`, který mně už tak přišel na hraně. Je to pro mě jako vytáhnout bokem „zbytečně“ nějakou konstantu nebo pomocnou funkci.